tests/createConnection.test.js

```diff
@@ -52,7 +52,7 @@ tap.test('Fastify register two different mongo instance', async t => {
     assert.ok(objectIdDocument)
 
     assert.throws(() => ObjectId(stringDocument._id), {}, { skip: false })
```

Comment:
```suggestion
    assert.throws(() => new ObjectId(stringDocument._id), {}, { skip: false })
```
Should also this get the `new` keyword before?